lib/ts/recipe/emailverification/recipe.tsx

```diff
@@ -67,6 +69,14 @@ export default class EmailVerification extends RecipeModule<
             );
             this.recipeImpl = builder.override(this.config.override.functions).build();
         }
+        if (this.config.mode !== "OFF") {
+            SessionClaimValidatorStore.addClaimValidatorFromOtherRecipe(
```

Comment:
We should add a test which enforces that the order of doing recipe init in the recipe list affects the priority of the redirection in case of claim validation failure.

Reply:
Will be done in a separate PR